cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -425,6 +425,8 @@ function(px4_nuttx_add_romfs)
 	endforeach()
 	add_custom_target(collect_extras DEPENDS ${extras})
 
+	string(REPLACE ";" "\ " gcc_c_flags_2 "${gcc_c_flags}")
```

Comment:
I needed to do this so that the arguments in the variable are written out as a string correctly. Not sure if I can move this that far up the cmake dependency, but I'll take a look.